examples/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -327,6 +327,7 @@ class _RecipePageState extends State<RecipePage> {
       scrollableKey: _scrollableKey,
       appBarBehavior: AppBarBehavior.scroll,
       appBar: new AppBar(
+        heroTag: 'nohero',
```

Comment:
Should we have a distinguished value (e.g., `null`) rather than using this random string, which might match some other hero tag that also uses that random string?